app/build.gradle

```diff
@@ -18,7 +18,7 @@ android {
         applicationId "com.willowtree.vocable"
         minSdkVersion 24
         targetSdkVersion 29
-        versionCode 25
+        versionCode 26
```

Comment:
Making sure this was intentional :)

Reply:
It is, it's carryover from another PR, just got the commits overlapped a little.  Once the other PR is merged this diff will go away, 26 is the current released version